### ISSUE TITLE
Render on 1 fixed camera angle (no rotation)

### ISSUE THREAD

**Issue by maris-otter** [johnferley/Game-Sprite-Creator#6]
Hello,

Unsure if this is an issue or just a setting i'm missing, but I'm trying to generate a spritesheet with only 1 camera angle - Side view only. My steps:
1. Loaded example scene example_output.blend
2. Add a Camera_Side preset, move it to the desired side view
3. Delete the existing camera and move Camera_Side into its place
4. Set no. of camera angles to 1
5. Render Sprites

![Output Parent_Character](https://user-images.githubusercontent.com/61036167/86522108-b8ff5f80-be26-11ea-8a3b-1ecebeadd57f.png)

The output is a spritesheet of the animations as desired but the single camera view is rotated slightly off of a straight side view. This rotation makes sense for the multi camera angle options, but shouldn't it not rotate at all for a single camera angle?

Please let me know if theres a setting i'm missing to accomplish this.

Thanks

**Comment by johnferley** [comment]
Sorry about taking so long to respond, I've been preoccupied with other projects.

I've tested this, and the issue is with the scene setup in my example scene. The Dimetric Parent is incorrectly rotated 315&#176; on the Z axis. When the program starts rendering, it sets the Z rotation to 0&#176;, which as a result causes the camera to be at a 45&#176; angle.

I've updated the example scene to fix this, however I've also noticed that the scene isn't correctly reset to as it was before the script started, which is probably what caused this problem in the first place. Once I've corrected that I'll release a fixed version.
